tap_pardot/client.py

```diff
@@ -41,7 +57,20 @@ class Client:
 
     def __init__(self, creds):
         self.creds = creds
-        self.login()
+        self.api_version = "4"
+        if self.has_oauth_values():
+            self.refresh_credentials()
+        elif self.has_api_key_auth_values():
+            self.login()
+        else:
+            raise AuthCredsMissingError("Requires OAuth credentials.")
```

Comment:
Shouldn't this say requires oauth credentials or "email, api key, and password" or whatever the other cred config values were?

Reply:
We originally said something along the lines of "Requires api key auth or oauth auth credentials (you have neither)" but this message treats api key auth more like a deprecated feature, which is closer to the truth.

Reply:
Sounds good, can you specify the config values mentioned for oauth then?